### ISSUE TITLE
Install Guide

### ISSUE THREAD

**Issue by MikeS159** [Venthe/vscode-fauxpilot#30]
The requirements for this plugin aren't obvious. I have Fauxpilot running. For this extension to work do you first need the copilot extension installed?

**Comment by nickthecook** [comment]
Yes, some setup instructions would be helpful.

**Comment by Venthe** [comment]
All you need is to have an address for the [fauxpilot server](https://github.com/moyix/fauxpilot). This extension only dependency is the server URL, which is set as the default Fauxpilot server adress.

I'll do a small write-up later

**Comment by YanouHD** [comment]
I though I was dumb but the lack of install guide is a pain, especially for people who runs Fauxpilot on a remote server. I guess the default address for the client is localhost:5000 ?

**Comment by Venthe** [comment]
Yeah, see https://github.com/Venthe/vscode-fauxpilot/blob/main/package.json#L53

**Comment by LillyWho** [comment]
The default configuration doesn't even result in a request being made to the server for me.
